### PR TITLE
UI: Show each asset USD/EUR/XXX value in the expanded ethereum accounts

### PR DESCRIPTION
**Pull request by LefterisJP**

### CODE REVIEW

File: electron-app/src/store/balances/getters.ts [hunk 1]
```diff
@@ -121,13 +122,29 @@ export const getters: GetterTree<BalanceState, RotkehlchenState> = {
     if (!ethAccount || isEmpty(ethAccount)) {
```
Comment (kelsos) [inline electron-app/src/store/balances/getters.ts]: FYI: I can't see the `accountTokens` getter used anywhere else in the code. It seems that it has been replaced with `accountAssets`. Doing a quick search returns only the declaration. Can you also verify that there is no usage? If it is not used then we can remove it.

File: electron-app/src/components/settings/AccountAssetBalances.vue [hunk 2]
```diff
@@ -31,30 +41,36 @@ import { Component, Prop, Vue } from 'vue-property-decorator';
...
+const { mapGetters: mapBalancesGetters } = createNamespacedHelpers('balances');
 
 @Component({
   components: { CryptoIcon },
   computed: {
-    ...mapGetters(['accountTokens']),
-    ...mapSessionGetters(['floatingPrecision'])
+    ...mapGetters(['accountAssets']),
+    ...mapSessionGetters(['floatingPrecision', 'currency']),
+    ...mapBalancesGetters(['exchangeRate', 'hasTokens'])
```
Comment (kelsos) [inline electron-app/src/components/settings/AccountAssetBalances.vue]: Is the `hasTokens` used anywhere in this component?  I cannot see it anywhere else in the code or in the properties. If it is unused we can remove it from the mapping.

File: electron-app/src/store/balances/getters.ts (hunk 1 above)
Comment (LefterisJP) [reply electron-app/src/store/balances/getters.ts]: indeed

File: electron-app/src/components/settings/AccountAssetBalances.vue (hunk 2 above)
Comment (LefterisJP) [reply electron-app/src/components/settings/AccountAssetBalances.vue]: nowhere, copy paste error

**Comment by LefterisJP** [comment]
Very good cooperation @kelsos! Thank you.

**Pull request merged**